### PR TITLE
LRDOCS-8096 - Guest Checkout (revised)

### PR DESCRIPTION
**Pull request by justinchoi001**
I added RSTs and updated the landing page.

### CODE REVIEW

File: docs/commerce/2.x/en/store-administration/guest-checkout.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Once Guest Checkout has been enabled, in order for their orders to be processed, guests must enter their email address during the checkout process.
+
+![Guest must enter their email address to proceed.](./guest-checkout/images/01.png)
+
+Guests are then redirected to log in or create an account.
+
+![Guest must enter their email address to proceed.](./guest-checkout/images/03.png)
+
+Alternately, users can create a [site page](https://learn.liferay.com/dxp/7.x/en/site-building/creating_pages.html) with the _Guest Checkout Authentication_ widget. Users can generate this page and widget using a Health Check on the [Channels setting](../managing-a-catalog/creating-and-managing-products/channels/channels-reference-guide.md).
```
Comment (jrhoun) [inline docs/commerce/2.x/en/store-administration/guest-checkout.md]: > "Alternately, users can create a ..."

Alternately, from what? My best guess at what this sentence is implying is that there are two methods for enabling Guest Checkout - one is by enabling it in Channels settings, and another way is by adding a Guest Checkout Authentication widget? The contrast we're drawing here is unclear to me.

File: docs/commerce/2.x/en/store-administration/guest-checkout.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+When a Guest submits an order, it is processed the same as any other order. See [Processing an Order](../orders-and-fulfillment/orders/processing-an-order.md) for more information.
+
+```tip::
+   Guests do not share order histories with other authenticated users, even if they have the same email addresses.
+```
+
+## Additional Information
+
+* [Enabling Guest Checkout](./enabling-guest-checkout.md)
+* [Introduction to Channels](../managing-a-catalog/creating-and-managing-products/channels/introduction-to-channels.md)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/store-administration/guest-checkout.md]: Broken link.

File: docs/commerce/2.x/en/store-administration/enabling-guest-checkout.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+To enable Guest Checkout:
+
+1. Navigate to the _Control Panel_ &rarr; _Commerce_ &rarr; _Channels_.
+1. Click on the desired channel.
+1. On the _General_ tab's _Orders_ section, slide the _Guest Checkout_ toggle to Enabled.
+1. Click _Save_.
+
+    ![Enable Guest Checkout](./enabling-guest-checkout/images/01.png)
+
+## Adding an Optional Authentication Widget
```
Comment (jrhoun) [inline docs/commerce/2.x/en/store-administration/enabling-guest-checkout.md]: Why? What's the purpose of this widget?

File: docs/commerce/2.x/en/store-administration/enabling-guest-checkout.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+1. On the _General_ tab's _Health Checks_ section, click _Fix Issue_ next to _Guest Checkout Authentication_. This creates a new authentication page with an option to sign in as Guest.
+
+    ![Add authentication page](./enabling-guest-checkout/images/02.png)
+
+1. Click _Save_ again when finished.
+
+Guest Checkout has been enabled.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/store-administration/enabling-guest-checkout.md]: So if I only follow these steps (but not the first set of steps) -- will I have enabled guest checkout?

**Comment by jrhoun** [comment]
Please see my questions and update accordingly @justinchoi001. Let me know if you have questions!

File: docs/commerce/2.x/en/store-administration/guest-checkout.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/store-administration/guest-checkout.md]: Revised.

**Pull request closed without merging**